packages/builder/src/pages/builder/app/[application]/design/_components/NewScreen/DatasourceTemplateRow.svelte

```diff
@@ -0,0 +1,42 @@
+<script>
+  import { Icon } from "@budibase/bbui"
+  export let datasource
+  export let selected = false
+
+  $: icon = datasource.type === "viewV2" ? "Remove" : "Table"
+</script>
+
+<div class="data-source-entry" class:selected on:click>
+  <Icon name={icon} color="var(--spectrum-global-color-gray-600)" />
+  {datasource.name}
+  {#if selected}
+    <span class="data-source-check">
+      <Icon size="S" name="CheckmarkCircle" />
+    </span>
+  {/if}
+</div>
+
+<style>
+  .data-source-entry {
+    cursor: pointer;
+    grid-gap: 12px;
```

Comment:
Shouldn't we use "space" for gaps like this?

Reply:
I actually just moved this into a new file rather than writing it, but in general yes it's better to use those variables. The designs don't usually use those variables though, so it's sometimes easier to use pixels to ensure we're matching the designs. I used to be a real stickler about this myself, but now I don't really mind.